launcher/net/PasteUpload.cpp

```diff
@@ -8,8 +44,22 @@
 #include <QJsonDocument>
 #include <QFile>
 
-PasteUpload::PasteUpload(QWidget *window, QString text, QString url) : m_window(window), m_uploadUrl(url), m_text(text.toUtf8())
+std::array<PasteUpload::PasteTypeInfo, 4> PasteUpload::PasteTypes = {
+    {{"0x0.st", "https://0x0.st", ""},
+     {"hastebin", "https://hst.sh", "/documents"},
```

Comment:
Can you make the text say the domain `hst.sh` like the others?

Reply:
> Can you make the text say the domain `hst.sh` like the others?

The point of that is to specify the type of pastebin, not the domain name of a specific instance. The reason the other ones are domain names is because the names aren't really unique enough and the service itself is better known by the name of the official instance.

Reply:
Fair enough.